Civi/API/Subscriber/ChainSubscriber.php

```diff
@@ -90,9 +90,11 @@ protected function callNestedApi($apiKernel, &$params, &$result, $action, $entit
       $result = ['values' => [0 => $oldResult]];
     }
 
+    // Keys which should not be assumed to be chain calls
+    $blacklist = ['api.has_parent', 'api_params'];
     // Scan the params for chain calls.
     foreach ($params as $field => $newparams) {
-      if ((is_array($newparams) || $newparams === 1) && $field <> 'api.has_parent' && substr($field, 0, 3) == 'api') {
+      if ((is_array($newparams) || $newparams === 1) && !in_array($field, $blacklist) && substr($field, 0, 3) == 'api') {
```

Comment:
ideally use TRUE as the 3rd parameter here (strict comparison)

Reply:
Ok, done.